nix/status-go/default.nix

```diff
@@ -3,14 +3,17 @@
 
 let
   inherit (stdenv.lib)
-    catAttrs concatStrings fileContents importJSON makeBinPath
+    catAttrs concatStrings concatStringsSep fileContents importJSON makeBinPath
     optional optionalString strings attrValues mapAttrs attrByPath
     traceValFn;
 
+  envFlags = callPackage ../tools/envParser.nix { };
```

Comment:
Would be nicer if it worked as a method that takes the `.env` file path as an argument to parse it. But it's fine.